src/ResourceManagement/AzureBackup/BackupServices.Tests/app.config

```diff
@@ -1,30 +1,59 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <appSettings>
-    <add key="ResourceName" value="backuprn" />
-    <add key="ResourceGroupName" value="backuprg" />
-    <add key="SubscriptionId" value="f5303a0b-fae4-4cdb-b44d-0e4c032dde26" />
-    <add key="ResourceId" value="5089188015289137043" />
-    <add key="Location" value="West US" />
-    <add key="ARMResourceId" value="/subscriptions/f5303a0b-fae4-4cdb-b44d-0e4c032dde26/resourceGroups/backuprg/providers/Microsoft.Backup/BackupVault/backuprn" />
-    <add key="ResourceType" value="Microsoft.Backup/BackupVault"/>
-    <add key="LocationShort" value="westus"/>
-    <add key="ContainerId" value="33"/>
-    <add key="ContainerStampId" value="eaf33dbc-a5b5-411a-a6af-1976a3f94602"/>
-    <add key="ContainerStampUri" value="https://sea-dev01-prot-4bodw.ext.trafficmanager.net"/>
-    <add key="ContainerFriendlyName" value="DPMDRSCALEINT1.DPMDOM02.SELFHOST.CORP.MICROSOFT.COM"/>
-    <add key="ContainerUniqueName" value="DPMDRSCALEINT1.DPMDOM02.SELFHOST.CORP.MICROSOFT.COM"/>
-  </appSettings>
-  <system.web>
-    <membership defaultProvider="ClientAuthenticationMembershipProvider">
-      <providers>
-        <add name="ClientAuthenticationMembershipProvider" type="System.Web.ClientServices.Providers.ClientFormsAuthenticationMembershipProvider, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" serviceUri="" />
-      </providers>
-    </membership>
-    <roleManager defaultProvider="ClientRoleProvider" enabled="true">
-      <providers>
-        <add name="ClientRoleProvider" type="System.Web.ClientServices.Providers.ClientRoleProvider, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" serviceUri="" cacheTimeout="86400" />
-      </providers>
-    </roleManager>
-  </system.web>
+	<appSettings>
+		<add key="ResourceName" value="backuprn" />
```

Comment:
We have done VS indentation this time. Going forward we will be using VS indentation.